Automattic/woocommerce-payments
# Submitting instant deposits using wp.data

Fixes #1140

#### Changes proposed in this Pull Request

* Moves submission of instant deposit to use wp.data
* After instant deposit is submitted and result is received, modal will close and a snackbar with a success/error notice will be shown.
* Overview and List will also refresh their data.

#### Testing instructions

1. You will need to use a server branch that has instant deposits available, the branch in 648-gh-Automattic/woocommerce-payments-server is the best to use, unless it has been merged.
2. Perform a transaction greater than $50 through your store with card number `4000000000000077`, which is [found in the Stripe docs](https://stripe.com/docs/testing#cards-responses).
3. Go to Payments > Deposits in the admin, the Instant Deposit button should appear.
4. Click the button to open the modal, then the button to _Deposit $xx now_.
5. Modal should close and show a snackbar notifying you the funds are in transit. The Overview and List will also reload.
6. The snackbar will have a link that will link you directly to the details of the deposit.

-------------------
